packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendar.h

```diff
@@ -20,12 +20,14 @@
 #import <Backpack/SimpleDate.h>
 #import <React/RCTViewManager.h>
 
+@class "RCTBPKColorBucket.h";
 @interface RCTBPKCalendar : BPKCalendar
 
 @property(nonatomic, copy) RCTBubblingEventBlock onDateSelection;
 @property(nonatomic, strong, readonly) NSCalendar *utcCalendar;
 
 @property(nonatomic, nullable) NSDate *rct_minDate;
 @property(nonatomic, nullable) NSDate *rct_maxDate;
+@property(nonatomic, nullable) NSArray<RCTBPKColorBucket *> *rct_colorBuckets;
```

Comment:
Can we specify `strong` on this and the other properties that don't specify it or `copy`?